### PR TITLE
[WIP] add new sprinter rok 

### PR DESCRIPTION
**Pull request by kamenjan**

### CODE REVIEW

File: content/rok_arih.rst [hunk 1]
```diff
@@ -0,0 +1,17 @@
+Rok Arih
+########
+
+:date: 2016-11-14 16:20
```
Comment (JanLikar) [inline content/rok_arih.rst]: Fix the date

File: content/rok_arih.rst [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+:tags: sprinters
+:category: Sprinters
+:slug: rok-arih
+:summary: Technical assitance
```
Comment (JanLikar) [inline content/rok_arih.rst]: Typo

File: content/rok_arih.rst [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+:image: /images/jcerjak.jpeg
+
+Python developer, enthusiastic about Open Source and specializing in web development, mainly with technologies Pyramid/Kotti, Flask and Plone CMS. Co-owner of Termitnjak d.o.o. In spare time he likes to dabble with vintage analog synthesizers.
```
Comment (JanLikar) [inline content/rok_arih.rst]: You forgot about changing this.

File: content/rok_arih.rst [hunk 4]
```diff
@@ -0,0 +1,17 @@
+Rok Arih
+########
+
+:date: 2016-11-14 16:20
+:tags: sprinters
+:category: Sprinters
+:slug: rok-arih
+:summary: Technical assitance
+:image: /images/jcerjak.jpeg
```
Comment (JanLikar) [inline content/rok_arih.rst]: Add your image, please

**Review by JanLikar — CHANGES_REQUESTED** [review]

**Comment by kamenjan** [comment]
Hey, I used my coworkers template, that's why WIP. Will fix it asap.

**Comment by JanLikar** [comment]
Sure, no problem :)

**Comment by kamenjan** [comment]
Please check for update, let me know if anything needs refactoring.

**Comment by JanLikar** [comment]
Ok, great!

**Review by JanLikar — APPROVED** [review]

**Pull request merged**